vela-templates/registry/auto-gen/gitops.yaml

```diff
@@ -0,0 +1,60 @@
+# Code generated by KubeVela templates. DO NOT EDIT.
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  name: gitops
+  namespace: vela-system
+  annotations:
+    definition.oam.dev/description: "gitops can fetching, building, updating and applying Kustomize manifests from git repo."
+spec:
+  workload:
+    definition:
+      apiVersion: source.toolkit.fluxcd.io/v1beta1
+      kind: GitRepository
+  schematic:
+    cue:
+      template: |
+        output: {
+        	apiVersion: "source.toolkit.fluxcd.io/v1beta1"
+        	kind:       "GitRepository"
+        	metadata: {
+        		name: context.name
+        	}
+        	spec: {
+        		interval: parameter.interval
+        		url:      parameter.repoUrl
+        		ref: branch: parameter.branch
+        	}
+        }
+        outputs: kustomize: {
+        	apiVersion: "kustomize.toolkit.fluxcd.io/v1beta1"
+        	kind:       "Kustomization"
+        	metadata: {
+        		name: context.name
+        	}
+        	spec: {
+        		interval: parameter.interval
+        		sourceRef: {
+        			kind: "GitRepository"
+        			name: context.name
+        		}
+        		path:       parameter.path
+        		prune:      true
+        		validation: "client"
+        	}
+        }
+        
+        parameter: {
+        	//+usage=The repository URL, can be a HTTP/S or SSH address.
+        	repoUrl: string
+        
+        	//+usage=The interval at which to check for repository updates.
+        	interval: *"1m" | string
+        
+        	//+usage=The Git reference to checkout and monitor for changes, defaults to master branch.
+        	branch: *"master" | string
+        
+        	//+usage=Path to the directory containing the kustomization.yaml file, or the set of plain YAMLs a kustomization.yaml should be generated for.
+        	path: string
```

Comment:
can we specify many path

Reply:
kustomize CRD only provide one path to set the directory containing the kustomization.yaml file
https://fluxcd.io/docs/components/kustomize/kustomization/